_config.yml

```diff
@@ -29,8 +29,8 @@ baseurl: "" # the subpath of your site, e.g. /blog
 # Twitter handle. Only the handle, not the URL.
 twitter: 18F
 
-# Configuration for Google Analytics, add your UA code here:
-google_analytics_ua: UA-????????-??
+# Configuration for Google Analytics, add your UA code here and uncomment
+# google_analytics_ua: UA-????????-??
```

Comment:
i know it's outside of the code change, but should we enable DAP by default?  should GSA be the default agency? will sites' analytics be recorded in DAP under GSA if a non GSA site does not update/remove this? is that an issue?